data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java

```diff
@@ -83,27 +83,47 @@ public static List<DataFile> listPartition(
     return listPartition(partition, uri, format, spec, conf, metricsConfig, mapping, 1);
   }
 
+  /**
+   * Returns the data files in a partition by listing the partition location using some number of
```

Comment:
Could we improve this description? It's not quite clear what this method is actually doing

Reply:
You're right.